tools/noise/subr.sh

```diff
@@ -299,6 +299,9 @@ get_image_iso() {
 	if [ -z "$iso" -o "$iso" = "65535" ]; then
 		iso=$(get_exif_key "$file" Exif.Photo.StandardOutputSensitivity)
 	fi
+	if [ -z "$iso" -o "$iso" = "65535" ]; then
```

Comment:
> which tag should be created

I guess you meant "in which order should the tags be evaluated".
But  yeah, for this, probably safe approach is better.